### PR TITLE
manually created call-to-action

### PR DESCRIPTION
**Pull request by Rizziepit**

### CODE REVIEW

**Comment by Rizziepit** [comment]
@miltontony ready for review. It first adds the Ebola content section and then implements the call-to-action links for FFL and Ebola.

File: development.ini [hunk 1]
```diff
@@ -5,7 +5,10 @@ unicore.content_repos =
     unicore_frontend_straighttalk_za
     unicore_frontend_ureport_za
     unicore_frontend_mariestopes_za
-    unicore_frontend_ffl_za
+
+iogt.content_section_url_overrides =
+    ffl = http://za.ffl.qa-hub.unicore.io/
```
Comment (miltontony) [inline development.ini:10]: This is going to be a problem. We'll need to link to to `/section/ffl/` rather than outside of the app.

Comment (Rizziepit) [inline development.ini:10]: From my discussion with Jonathan, the call-to-action blocks are external links to the individual websites.

**Comment by miltontony** [comment]
:+1:  happy to land this and await the outcome of the FB decision on linking. But I'd rather Unicef finalize the content on those 2 repositories and we won't need this manual call to action bizniz

**Comment by Rizziepit** [comment]
Thanks @miltontony.

I'm going to land this, but we can revert it very easily. I put all the call-to-action changes in a single commit here https://github.com/universalcore/springboard-iogt/commit/3071b496a9e807930755703d2bc69a3f056b0b36.

**Pull request merged**